package.json

```diff
@@ -35,6 +35,7 @@
     "react-tailwindcss-datepicker": "^1.6.6",
     "react-tooltip": "^5.21.5",
     "reflect-metadata": "^0.1.13",
+    "sharp": "^0.32.6",
```

Comment:
Unrelated, re-adding coz removed trying to fix images and it was an issue with Next.js which after upgrading to newer version got fixed.